test/liveUtils.js

```diff
@@ -0,0 +1,272 @@
+var promisify = require('promisify-node');
+var childProcess = require('child_process');
+var readlineSync = require('readline-sync');
+
+var Travis = require('travis-ci');
+var travis = new Travis({ version: '2.0.0' });
+
+travis.authenticate = promisify(travis.authenticate);
+travis.users.get = promisify(travis.users.get);
+travis.users.sync.post = promisify(travis.users.sync.post);
+
+var GitHub = require('github');
+var github = new GitHub({
+  version: '3.0.0',
+  protocol: 'https',
+  headers: {
+    'user-agent': 'Oghliner',
+  },
+});
+
+var liveUtils = {
+  repoName: 'test_oghliner_live' + process.version + process.pid,
```

Comment:
Nit: these repo names would be a bit easier to read with separator characters between the version and id, f.e.: `test_oghliner_livev0.12.729326` -> `test_oghliner_live_v0.12_729326`.